ehr2vec/model/model.py

```
@@ -2,7 +2,7 @@
 
 import torch
 import torch.nn as nn
-from transformers import BertModel
+from transformers import ModernBertModel
 from transformers.models.roformer.modeling_roformer import RoFormerEncoder
```

Ruff (0.8.2)

`transformers.models.roformer.modeling_roformer.RoFormerEncoder` imported but unused

Remove unused import: `transformers.models.roformer.modeling_roformer.RoFormerEncoder`

(F401)